OpenMEEG/libs/OpenMEEG/include/geometry_io.h

```diff
@@ -238,18 +238,16 @@ namespace OpenMEEG {
         // Add the array of Indices to polydata points
         polydata->GetPointData()->AddArray(point_indices);
         // Add optional potentials and currents
-        if (data.nlin() != 0)
+        if (data.nlin() != 0) {
             for (unsigned j = 0; j < data.ncol(); ++j) {
                 polydata->GetPointData()->AddArray(potentials[j]);
                 polydata->GetCellData()->AddArray(currents[j]);
             }
+        }
```

Comment:
This change has nothing to do with the pull request. And don't mention some coding rules whcih were not approved in the first time....
The {} actually diminish the readability by reducing the number of real code in a given vertical space. It is totally redundant with the indentation (which I strongly support for readability).
Finally, we already agreed that after the release we will use a tool to reformat once and for all the code (at commit or push time, I did that in some other repo, I do not remember the details). There are two good C++ beautifiers available... uncrustify is a nice tool to configure those.